Anlab.Core/Models/TestItemPrices.cs

```diff
@@ -35,7 +35,7 @@ public decimal SetupPrice
         {
             get
             {
-                if (Nonrep)
+                if (Nonrep && !Id.StartsWith("G-", StringComparison.OrdinalIgnoreCase))

```

Comment:
@jSylvestre seems ok, though we are doing this check in a lot of places.  Next time you come around to it maybe throw together a helper to figure out if the ID represents a group so we don't have to keep doing the startsWith